vulcan/lib/orchestration.rb

```diff
@@ -69,12 +86,20 @@ def command(storage:, input_files:, output_files:)
           "-v",
           "/var/run/docker.sock:/var/run/docker.sock:ro",
           "-v",
+          "/etna/packages/magby/magby/:/app/magby/:ro",
+          "-v",
           "#{Vulcan.instance.config(:archimedes_exec_volume)}:/archimedes-exec",
           Vulcan.instance.config(:archimedes_run_image),
           "poetry",
           "run",
           "archimedes-run",
-          "--isolator=docker",
+          "--isolator=docker"
+      ] + (is_dev? ? dev_options : []) + [
```

Comment:
this goes away once we take the approach mentioned before with baing magby into the image.

Reply:
Yeah, that would be great to get rid of the `--local-packages` flag. I think most of the `dev_options` stuff would go away once we fix the Magma HMAC stuff, too. Perhaps only adding it to the `monoetna_edge_net` network would be required at that point?